src/ff/prime_field.rs

```diff
@@ -67,7 +73,13 @@ macro_rules! field_impl {
 
         impl From<$field> for u8 {
```

Comment:
eh I don't think we should have those conversions defined for every field. In general, the only thing we could guarantee is conversion from field to the largest int (u128). Everything else is fallible in general (like values from $F_{2^{32}-5}$ field > 255 cannot be converted to `u8`. The convenience of doing just `u8::from($field)` comes with a steep price of runtime errors that are hard to debug and investigate. 

we could implement `TryInto` for them if we really want .

Reply:
I agree with Alex.  Though we could probably change this to u64 on the basis that we don't ever want something larger than that.